source/help/getting-started/setting-your-status-availability.rst

```diff
@@ -43,8 +43,21 @@ When you set your availability, an availability icon displays next to your name.
 
 Other members can see your availability anywhere they can see your name, such as the channel sidebar, within conversations, and within Direct Messages. 
 
+Do Not Disturb Timer
```

Comment:
@cwarnermm I think this section should be more general "Do Not Disturb". Start with an explanation of what DND is (ie set when you're unavailable or need to concentrate and it pauses all notifications including email, mobile and desktop), then say that in v5.36+ you can set DND for a specified period of time